app/models/external-accounts.ts

```diff
@@ -0,0 +1,15 @@
+import { attr } from '@ember-data/model';
+
+import OsfModel from './osf-model';
+
+export default class ExternalAccountsModel extends OsfModel {
+    @attr('string') provider!: string;
+    @attr('string') profileUrl?: string;
+    @attr('string') displayName!: string;
```

Comment:
Not sure if this will be a problem, but might be safer to use `@attr('fixstring')` for these.